$OpenBSD: patch-configure,v 1.3 2008/11/21 18:20:05 jasper Exp $

Force disable Java support. Configure script is broken and doesn't
correctly disable it.

--- configure.orig	Sun Nov 16 17:25:05 2008
+++ configure	Tue Nov 18 12:25:13 2008
@@ -12358,7 +12358,7 @@ fi
 
 # Java support
 
-
+if false; then # skip Java configure part, it's broken.
 # Check whether --with-java_home was given.
 if test "${with_java_home+set}" = set; then
   withval=$with_java_home; if test "$withval" = "no" ; then
@@ -12831,8 +12831,11 @@ else
   BUILD_JAVA_PACKAGE_FALSE=
 fi
 
+fi # skip java
 
-
+JAVA_SUPPORT="false"
+BUILD_JAVA_PACKAGE_TRUE='#'
+BUILD_JAVA_PACKAGE_FALSE=
 
 # GNOME Speech Java Package location
 
